src/views/layout/PageLayout/PageLayout.common.less

```diff
@@ -288,9 +256,56 @@
       background-color: #f3f3f3;
     }
   }
-
+  //unused
   .node-input-container {
     width: 100%;
     margin-right: 20px;
   }
 }
+.mac.wrap {
```

Comment:
Can you please rename this class?

It is named `mac` because there was a `windows` class in the past

A more generic name like `app-container` would be more appropriate and avoid confusion in the future

Reply:
> generic name like

Certainly.

I did this before, but @evias asked me to keep the purity of this pr. I will deal with what you said in the PR later